### ISSUE TITLE
how to install emscripten in windows 7_64?

### ISSUE THREAD

**Issue by appemit** [emscripten-core/emscripten#15028]
how to install emscripten in windows 7_64?

the default python 3.9 is not supported in windows7.

**Comment by kripken** [comment]
Closing as a duplicate of https://github.com/emscripten-core/emsdk/issues/892

Please do not open duplicate issues unless there is a specific reason, as they split the conversation and just cause duplicate emails to be sent to the same group of people in many cases.
